src/d_deh.c

```diff
@@ -123,6 +123,20 @@ char *s_D_CDROM     = D_CDROM;
 char *s_PRESSKEY    = PRESSKEY;
 char *s_PRESSYN     = PRESSYN;
 char *s_QUITMSG     = QUITMSG;
+char *s_QUITMSG1    = QUITMSG1;
```

Comment:
I wonder if there is a preprocessor macro that one could use in a loop? Do you know of something like that?

Reply:
It's a pain: https://stackoverflow.com/questions/319328/how-to-write-a-while-loop-with-the-c-preprocessor C preprocessor is for simple things 🤓

Reply:
🤣 no way.